modules/concept-docs/pages/buckets-and-clusters.adoc

```diff
@@ -8,6 +8,35 @@ The Couchbase C SDK provides an API for managing a Couchbase cluster programmati
 
 include::6.5@sdk:shared:partial$clusters-buckets.adoc[tag=management]
 
+== Creating and Removing Buckets
+
+To create or delete a bucket using the REST API :
+
+[source,c]
+----
+    lcb_CMDHTTP htcmd = {};
+    LCB_CMD_SET_KEY(&htcmd, path.c_str(), path.size());
+    htcmd.body = params.c_str();
+    htcmd.nbody = params.size();
+    htcmd.content_type = "application/x-www-form-urlencoded";
+    htcmd.method = LCB_HTTP_METHOD_POST;
+    htcmd.type = LCB_HTTP_TYPE_MANAGEMENT;
+    htcmd.username = "Administrator";
+    htcmd.password = "password";
+    lcb_http3(instance, NULL, &htcmd);
+    lcb_wait(instance);
+
+    // now remove the bucket
+    memset(&htcmd, 0, sizeof htcmd);
+    path = "/pools/default/buckets/newBucket";
+    LCB_CMD_SET_KEY(&htcmd, path.c_str(), path.size());
+    htcmd.method = LCB_HTTP_METHOD_DELETE;
+    htcmd.type = LCB_HTTP_TYPE_MANAGEMENT;
+    htcmd.username = "Administrator";
+    htcmd.password = "password";
+    lcb_http3(instance, NULL, &htcmd);
```

Comment:
Store code snippets in the external files, please. And make sure they compile without errors

Reply:
I've updated the file to reference the example from the devguide directory.